dotenv-config.js

```diff
@@ -1,3 +1,3 @@
 // dotenv doesn't provide an override option, so this will allow the .env.test to decide the environment
 process.env.NODE_ENV = 'test';
-require('dotenv').config({ path: './.env.test' });
\ No newline at end of file
+require('dotenv').config({ path: './.env.test' });
```

Comment:
was there a newline added or something? I can't see a difference

Reply:
Yeah, that's the change here.